.github/workflows/cmake.yml

```diff
@@ -688,6 +682,18 @@ jobs:
           Get-Command ccache
           (Get-Command ccache).Source
           ccache --show-stats --verbose --version
+      - name: Download Apache Arrow latest source archive and verify via PGP
+        run: |
+          $LATEST_RELEASE_INFO = Invoke-RestMethod "https://api.github.com/repos/apache/arrow/releases/latest"
+          # e.g. "apache-arrow-19.0.0"
+          $APACHE_ARROW_VERSION = $LATEST_RELEASE_INFO.tag_name
+          # "apache-arrow-19.0.0" -> "19.0.0"
+          $ARROW_VERSION = $APACHE_ARROW_VERSION -replace '^apache-', ''
+          $DOWNLOAD_URL = "https://dlcdn.apache.org/arrow/${ARROW_VERSION}/${APACHE_ARROW_VERSION}.tar.gz"
+          Invoke-WebRequest -Uri $DOWNLOAD_URL -OutFile "${APACHE_ARROW_VERSION}.tar.gz"
+          mkdir apache-arrow
+          tar -xzf "${APACHE_ARROW_VERSION}.tar.gz" --strip-components=1 -C apache-arrow
+          Remove-Item "${APACHE_ARROW_VERSION}.tar.gz"
```

Comment:
If we should verify the checksum here, please let me know.
ref: https://www.apache.org/dyn/closer.lua/arrow/arrow-19.0.0/

Reply:
We can omit it here.